.github/workflows/translations.yaml

```diff
@@ -0,0 +1,63 @@
+name: Automated syncronization and update of translations
+
+on:
+  shedule:
+    branches: [ master ]
+    - cron:  '0 0 */3 * *'
+
+jobs:
+  translations_sync_and_update:
+    name: Sync translations from Transifex and update source .ts files with new unstranslated strings
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: checkout repository
+      uses: actions/checkout@v2.3
+
+    - name: install necessary tools from Ubuntu repos
+      run: |
+        sudo apt update
+        sudo apt install \
+          qttools5-dev-tools qt5-qmake transifex-client
+
+    - name: Pull translations from Transifex
+      run: |
+        tx pull -a -f -r qbittorrent.qbittorrent_master
```

Comment:
just curious, is `-f` necessary?

Reply:
And would `--parallel` option help?

Reply:
`-f` because I don't want to have it abort some translations due to timestamps. In any case, git will detect only changed files. The extra bandwidth for these litle txt files isn't a problem.
I can add `--parallel`. I suppose we have few files so probably it won't create a problem.